livekit-agents/livekit/agents/utils/codecs/mp3.py

```diff
@@ -27,13 +31,10 @@ class Mp3StreamDecoder:
     """
 
     def __init__(self):
-        try:
-            globals()["av"] = import_module("av")
-        except ImportError:
+        if importlib.util.find_spec("av") is None:
             raise ImportError(
                 "You haven't included the 'codecs' optional dependencies. Please install the 'codecs' extra by running `pip install livekit-agents[codecs]`"
             )
```

Comment:
let's keep `import av` and catch the ImportError instead of `importlib.util.find_spec("av")`?

Reply:
That will make above import unused

Reply:
maybe you can use # noqa?